blog/_posts/2017-09-13-Blog-1team-Serv0101.md

```diff
@@ -0,0 +1,38 @@
+---
+title: “Blog- Team Serv0101”
+layout: post
+created_at: Wed Sept 13 2017
```

Comment:
Your blog post is scheduled on Sunday, September 17. Fix the date and day everywhere.